api/drivers/qemu/driver.js

```diff
@@ -40,15 +40,24 @@ class QemuDriver extends Driver {
    * @return {Promise}
    */
   initialize() {
-    return ConfigService.get('qemuServiceURL', 'qemuServicePort')
+    return ConfigService.get('qemuServiceURL', 'qemuServicePort', 'qemuDrive')
       .then((config) => {
-
-        let requestOptions = {
-          url: 'http://' + config.qemuServiceURL + ':' + config.qemuServicePort,
-          method: 'GET'
-        };
-
-        return request(requestOptions);
+        Image.update({
+          default: true
+        }, {
+          iaasId: 'Qemu',
+          name: config.qemuDrive,
```

Comment:
`qemuDrive` is the absolute path the qcow2 file. Why should it be the name of the default image.